src/util/report-util.js

```diff
@@ -159,6 +159,144 @@ export function generateCSV (params) {
   })
 }
 
+export function generateMultiplePDF (data) {
+  debug('generating PDF output...')
+  const dateFrom = data[0].dateFrom
+  const dateTo = data[0].dateTo
+  const formatDateWith = data[0].formatDateWith
+  const accountId = data[0].accountId
+  const docDefinition = {
+    info: {
+      title: `report`,
+      creationDate: new Date()
+    },
+
+    styles: {
+      title: { fontSize: 24, margin: [0, 0, 0, 10] },
+      heading1: { fontSize: 22, bold: true, margin: [0, 10, 0, 5] },
+      logo: { alignment: 'center', margin: [0, 0, 0, 25] }
+    },
+
+    content: [
+      { image: REPORT_LOGO, width: 200, style: 'logo' },
+      { text: `Report (${formatDateWith(dateFrom, 'MMM. D YYYY HH:mm')} - ${formatDateWith(dateTo, 'MMM. D YYYY HH:mm')})`, style: 'title' },
+      { text: `Account ID: ${accountId}` }
+    ]
+  }
+
+  data.forEach(params => {
+    const { formatDate, formatPrecision } = params
+    const data = generateReportData.call(this, { ext: 'pdf', ...params })
+    const walletDefinition = [
+      { text: `Wallet: ${data.walletName} (${data.cryptoCurrencyUnit})` },
```

Comment:
I think for wallet name we should use style: 'heading1' 
And for other titles - style: 'heading2'